cmd/raidz_test/raidz_test.c

```diff
@@ -839,7 +839,7 @@ static kcondvar_t sem_cv;
 static int max_free_slots;
 static int free_slots;
 
-static _Noreturn void
+static __attribute((noreturn)) void
```

Comment:
Shouldn't this have been `__attribute__` like everywhere else?

Reply:
Yup, fixed.